pkg/alpine/mirrors/3.16/main

```diff
@@ -88,13 +88,13 @@ libpciaccess-dev
 libpng
 libpng-dev
 librelp-dev
-libressl-dev
+libretls
```

Comment:
Also a TLS package?

Reply:
Well, seems it is not required anywhere now, will try to rebuild without this library.

Reply:
Removed